examples/Tutorials/Boilerplate/JustRide/ViewController.swift

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
I'm not sure of the benefit of an empty viewcontroller like this. Maybe we want to have some boilerplate of setting up a LoginManager or a RidesClient? We can sync offline about what you were thinking here